src/app/projects/new-project/new-project.component.html

```diff
@@ -1,22 +1,32 @@
 <div class="ui-g" id="papaya-box">
-  <div class="ui-g-6 ui-g-nopad">
+  <div class="ui-g-8 ui-g-nopad">
     <h5>Nowy projekt</h5>
 
     <div class="ui-g-6">
       <p>Nazwa:</p>
-      <input type="text" pInputText [(ngModel)]="project.name"/>
+      <input class="papaya-input" type="text" pInputText [(ngModel)]="project.name"/>
     </div>
 
     <div class="ui-g-6">
       <p class="first">Współczynnik początkowy: {{project.initialCoefficient}}</p>
-      <p-slider [(ngModel)]="project.initialCoefficient" [style]="{'width':'15em'}"
+      <p-slider [(ngModel)]="project.initialCoefficient" [style]="{'width':'80%'}"
```

Comment:
to może być po prostu ```style="width: 80%"```

Reply:
otóż nie.
![Zrzut ekranu (603)](https://user-images.githubusercontent.com/26676749/57465146-4c1e2300-727e-11e9-98d4-e8d44cda8093.png)

Reply:
a wklej kod

Reply:
`<div class="ui-g-6">
      <p class="first">Współczynnik początkowy: {{project.initialCoefficient}}</p>
      <p-slider [(ngModel)]="project.initialCoefficient" style="width: 80%"
                [min]="1" [max]="3" [step]="0.05">
      </p-slider>
    </div>`

Reply:
no okej, zwracam honor